src/__test__/calculate.test.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { render, fireEvent, screen } from '@testing-library/react';
+import Calculator from '../components/Calculator';
+
+test('renders calculator', () => {
+  const { container } = render(<Calculator />);
+  expect(container).toMatchSnapshot();
+});
+
+test('clicking AC button resets the display to "0"', () => {
+  const { container } = render(<Calculator />);
+  fireEvent.click(screen.getByText('AC'));
+  expect(container).toMatchSnapshot();
+});
+
+test('clicking number buttons updates the display', () => {
+  const { container } = render(<Calculator />);
+  fireEvent.click(screen.getByText('1'));
+  fireEvent.click(screen.getByText('2'));
+  expect(container).toMatchSnapshot();
+});
```

Comment:
- [ ] @Hsuwaddyhtat22 as previously requested kindly ensure your tests cover all functionalities and scenarios of the calculate.js module, they should be 6 in total.